torch/fx/experimental/proxy_tensor.py

```diff
@@ -713,7 +728,8 @@ def can_handle_tensor(x: Tensor) -> bool:
             # Adding an undefined attribute to Tensor?
             args[0].proxy = proxy_out  # type: ignore[attr-defined]
 
-    out = func(*args, **kwargs)
+    with enable_thunkify(proxy_mode.tracer):
+        out = func(*args, **kwargs)
```

Comment:
Thunkify any of the internal sizey compute that happens. Actually, this shouldn't really make a difference because sym dispatch tracing is disabled at this point in time, but whatever, doesn't hurt.